minimal/libp2p.go

```diff
@@ -30,19 +31,26 @@ func (s *Server) setupLibP2P() error {
 	}
 	s.addrs = []ma.Multiaddr{addr}
 
+	bwc := metrics.NewBandwidthCounter()
```

Comment:
What does this do?

Reply:
Sorry, it's caused by my misunderstanding. Used this as an indicator to get good peer from closest peers